graph/src/data/subgraph/mod.rs

```diff
@@ -268,8 +267,6 @@ pub enum SubgraphRegistrarError {
     NameNotFound(String),
     #[error("Ethereum network not supported by registrar: {0}")]
     NetworkNotSupported(String),
-    #[error("Ethereum nodes for network {0} are missing the following capabilities: {1}")]
```

Comment:
I understood that you moved the capabilities code to the chain/ethereum crate, but this won't happen anymore?

I'm a bit confused.

Reply:
It really is confusing, because this code was just already dead.

Reply:
Oh! okay 🙂